tests/inspectdb/tests.py

```diff
@@ -299,3 +299,16 @@ def test_introspection_errors(self):
         self.assertIn("# Unable to inspect table 'nonexistent'", output)
         # The error message depends on the backend
         self.assertIn("# The error was:", output)
+
+    def test_indexes_meta(self):
+        out = StringIO()
+        call_command('inspectdb', table_name_filter=lambda tn: tn.startswith('inspectdb_indexes'), stdout=out)
+        output = out.getvalue()
+        self.assertIn('        indexes = [', output, msg='inspectdb should generate indexes of the models.')
+        # Single column index via db_index and auto-generated name
```

Comment:
No, models file created by inspectdb doesn't have `db_index`. I'll remove this comment as it is confusing.
It's not an error to specify `db_index=True` and also specify single column index in `Meta.indexes` but it is not a good thing to create multiple indexes on the same column (performance wise at db level) .